method/link.js

```diff
@@ -70,13 +71,15 @@ module.exports = function Link (ssb) {
         feedGroupLink.read(linkId, cb)
       })
     },
-    createSubGroupLink ({ group, subGroup }, cb) {
+    createSubGroupLink ({ group, subGroup, admin }, cb) {
       const input = {
         parent: group,
         child: subGroup,
         recps: [group]
       }
 
+      if (admin) input.admin = true
```

Comment:
and this

Reply:
only though is to have a step which removes empty fields (`undefined`) inside that create method

Reply:
actually, I think when it gets encrypted `undefined` fields will get dropped. Also, we have schema checkers and if something like undefined made it through it would probably error before publishing right?

in which case we can teach `ssb-crut` to prune off all empty fields

Reply:
Yup :)